calibrators/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/board_detections/board_detection.py

```diff
@@ -232,3 +238,27 @@ def get_speed(self, last: "BoardDetection") -> float:
         last_image_points = last.get_flattened_image_points()
 
         return np.linalg.norm(current_image_points - last_image_points, axis=-1).mean()
+
+    def get_aspect_ratio_pattern(self, model: CameraModel) -> float:
+        """Get aspect ratio using the calibration pattern, which should be squared."""
+        tilt, pan = self.get_rotation_angles(model)
+        acceptance_angle = 10
+
+        # dont update if we the detection has big angles, calculation will not be accurate
```

Comment:
grammar

Reply:
thanks, i will fix it

Reply:
f2ad5cce4cffc72284d0b575020c40851475872e

Reply:
This comment does not have the changes related to this conversation...